nautobot/extras/templates/extras/customfield.html

```diff
@@ -53,6 +53,10 @@
                         <td>Filter Logic</td>
                         <td><span>{{ object.filter_logic | capfirst }}</span></td>
                     </tr>
+                    <tr>
+                        <td>Hide From Detail Tab</td>
```

Comment:
I'm not thrilled with this wording, but I'm having a hard time coming up with anything better to suggest.

Reply:
OK I'll leave it until we have a better name!

Reply:
Maybe "Move to Advanced Tab" might be better? Not sure.